src/Stratis.Bitcoin/Interfaces/IBlockDownloadState.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Stratis.Bitcoin.Interfaces
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.Interfaces
 {
     public interface IBlockDownloadState
     {
+        /// <summary>
+        /// Checks whether the node is currently in the process of initial block download.
+        /// </summary>
+        /// <returns><c>true</c> if the node is currently doing IBD, <c>false</c> otherwise.</returns>
         bool IsInitialBlockDownload();
+
+        /// <summary>
+        /// Sets last IBD status update time and result.
+        /// <para>Used in tests only.</para>
+        /// </summary>
+        /// <param name="val">New value for the IBD status, <c>true</c> means the node is considered in IBD.</param>
```

Comment:
Can name this properly? Like blockDownloadStatus or something.

Reply:
Fixed.